### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jdonszelmann**
just thought this'd be a nice one

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
[the remark this answers is not in the document]
Comment (jdonszelmann) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fuck these

File: .codeclimate.yml [hunk 2]
```diff
@@ -14,3 +14,4 @@ plugins:
     enabled: true
 exclude_patterns:
   - "LICENSE.md"
+  - ".github/"
```
Comment (dsluijk) [inline .codeclimate.yml:17]: Why should this be excluded? These are markdown files which should be checked

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (dsluijk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe add a default label `bug`?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (dsluijk) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Maybe add a default label bug?

**Review by dsluijk — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 4 above)
Comment (NULLx76) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Default label for a feature request should probably not be bug but enhancement instead right?

Comment (dsluijk) [reply .github/ISSUE_TEMPLATE/feature_request.md]: yes, sorry, I copy-pasted it from the bug template

File: .codeclimate.yml (hunk 2 above)
Comment (jdonszelmann) [reply .codeclimate.yml:17]: only includes bug report templates which are incorrectly flagged as having incorrect indentation while it's fine

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (NULLx76) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: It's probably clearer if there is a newline between heading and content. This applies to all the headings

**Review by NULLx76 — CHANGES_REQUESTED** [review]

**Review by dsluijk — DISMISSED** [review]

**Comment by dsluijk** [comment]
Can this be closed or will it be finished at some point?

**Review by NULLx76 — APPROVED** [review]

**Pull request merged**